torch/fx/passes/split_module.py

```diff
@@ -412,13 +418,26 @@ def instantiate_node_partition_mapping(node):
     # add placeholders to partition inputs
     for partition_name in sorted_partitions:
         partition = partitions[partition_name]
+        new_inputs: Dict[str, None] = {}
         for inp in partition.inputs:
-            placeholder = partition.graph.placeholder(
-                inp,
-                type_expr=orig_nodes[inp].type,
-            )
+            orig_node = orig_nodes[inp]
+            # We don't pass in get_attr nodes as inputs to the partition, but
+            # instead set them as targets and use getattr within the module
+            if orig_node.op == "get_attr":
+                assert isinstance(orig_node.target, str)
+                placeholder = partition.graph.get_attr(orig_node.target)
+                partition.targets[orig_node.target] = _get_attr_from_qualname(
```

Comment:
i don't recall what the .targets property of a graph means.  is it a list of all the things that the graph accesses via .getattr?

Reply:
oh, does this mean that we set an attribute on partition so that we can access partition.name instead of accessing parent_mod.name?

Reply:
It's just the attributes that are assigned to the graphmodule. So it's a `Dict` from attribute name to value.